src/main.js

```diff
@@ -29,7 +29,7 @@ export function createG4U (element, mapConfig, layerConfig, modules) {
       $(element).empty()
 
       // for remote analysis and debugging - not used inside of the software
-      window.map = new G4UMap(element, mapConfig, layerConfig, {
+      window.map = new G4UMap(element, window.clientConfPath || clientConfPath, window.layerConfPath || layerConfPath, {
```

Comment:
I'd suggest to name the global variables g4uClientConfPath and g4uLayerConfPath so their meanings are more clear in an environment where is more than just the map in the DOM.